File: LineUp/python/LineUp/module/lumodule.py
class LuModule:
    def __init__(self):
        self.class_data = {}

    def add_class(self, name, class_data):
        setattr(self, name, class_data)
        self.class_data[name] = getattr(self, name)


class ModuleSystem:
    def __init__(self):
        self.module_name = {}

    def add_modules(self, name):
        setattr(self, name, LuModule())
        self.module_name[name] = getattr(self, name)

    def add_class(self, module_name, class_name, class_data):
        self.module_name[module_name].add_class(class_name, class_data)

File: LineUp/php/LineUp/module/LuModule.php
<?php

/**
 * Description of LuModule
 *
 * @author MisterMine01
 */
class LuModule {
    public function __construct() {
    }
    
    public function add_class($name, $class_data) {
        $this->$name = $class_data;
    }
}

class ModuleSystem {
    public function __construct() {
    }
    
    public function add_module($name) {
        $this->$name = new LuModule();
    }
    
    public function add_class($name, $class_name, $class_data) {
        $this->$name->add_class($class_name, $class_data);
    }
}